scripts/create_framework.sh

```diff
@@ -71,23 +71,23 @@ xcodebuild -create-xcframework -framework './SmartDeviceLink-Device.xcarchive/Pr
 
 # TODO - is there a way we can test that the build was successful.
 
-folder="SmartDeviceLink.xcframework"
+framework_folder="SmartDeviceLink.xcframework"
 zip_file_name="SmartDeviceLink-$new_version_number.xcframework.zip"
 # Kill the old zip if present. Useful for re-running the script
 if [ -f $zip_file_name ]; then rm $zip_file_name; fi
-# Verify folder exists before acting on it.
-if [ -d "$folder" ]; then 
-    zip $zip_file_name $folder
+# Verify framework_folder exists and create a zip from it.
+if [ -d "$framework_folder" ]; then 
+    tar -cf $zip_file_name $framework_folder
```

Comment:
<img width="224" alt="Screen Shot 2022-08-02 at 2 23 33 PM" src="https://user-images.githubusercontent.com/1767063/182446452-02cc50e8-684b-4f1a-b13a-a96b98874830.png">

No change :(

Reply:
We need to look closer at this.  In my testing the script did produce a good zip.  A diff of the contents to the original shows no changes.  They are identical.  I also tested manually running the command and right click-compress.